### ISSUE TITLE
consider splitting content by usage

### ISSUE THREAD

**Issue by oliver-sanders** [cylc/cylc-doc#449]
See https://github.com/cylc/cylc-doc/pull/422#discussion_r822434435

Consider splitting the docs:

* For frontend users.
  * client commands
  * user interfaces
* For workflow developers.
  * user guide
  * workflow design guide
* For administrators.
  * advanced installation
  * conda recipes
  * system configuration
* For developers.
  * writing plugins

**Comment by hjoliver** [comment]
Makes sense. There's not much overlap between the categories, so not too difficult to do.

**Comment by oliver-sanders** [comment]
Here are some of the things I think should go in the "administrator guide" section:

* [Site installation instructions](https://cylc.github.io/cylc-doc/stable/html/installation.html#advanced-installation).
* [Most of the scheduler configuration section]((https://cylc.github.io/cylc-doc/stable/html/user-guide/writing-workflows/scheduler.html) (the event handlers bit can stay in the user guide).
* [Cylc UI Server authorisation](https://cylc.github.io/cylc-doc/stable/html/user-guide/sharing-access-to-workflows.html).
* Possibly the [client-server interaction bit](https://cylc.github.io/cylc-doc/stable/html/user-guide/running-workflows/authentication-files.html) which isn't really something users need to know.
* https://github.com/cylc/cylc-doc/issues/173
* Any Jupyter Hub stuff like https://github.com/cylc/cylc-doc/issues/36
